x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/predicate/operator/comparison/In.java

```diff
@@ -177,6 +177,10 @@ protected TypeResolution resolveType() {
         return super.resolveType();
     }
 
+    public TypeResolution validateInTypes() {
```

Comment:
I've added this one because I didn't want to change the `resolveType` method and to be able to call it from the optimizer rule.